sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -249,6 +253,17 @@ abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
    */
   object Aggregation extends Strategy {
     def apply(plan: LogicalPlan): Seq[SparkPlan] = plan match {
+      case Deduplication(keys, child) =>
```

Comment:
Shouldnt there be a new strategy? After all dropping duplicates is not conceptually an aggregation. its just so happens that it can be implemented as a aggregation?